contracts/test/UniswapV2RouterStub.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity <0.9.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
```

Comment:
do we need this import?

Reply:
Nope. Done in https://github.com/keep-network/coverage-pools/pull/66/commits/cd57620af198ce557ef8d935bd042ab64be035d1